src/lib/markets/index.ts

```diff
@@ -322,6 +323,12 @@ export class MarketsModule {
           orderDetails,
         })
       },
+      /**
+       * Merge multiple build transaction data objects into a single object.
+       * Can be used to batch multiple operations into a single transaction.
+       * @param transactions Array of transaction data objects.
+       */
+      mergeMultiInvokerTxs: mergeMultiInvokerTxs,
```

Comment:
Now that I think of it, since the multiinvoker covers more than just markets we should leave this out of any specific module and just use it as a util